gitlog+jira/src/main/java/org/bsc/reporting/renderer/GitLogJiraIssuesRenderer.java

```diff
@@ -46,21 +55,28 @@ public GitLogJiraIssuesRenderer(
         this.jiraProjectKeyList = jiraProjectKeyList;
         this.gitLogTagNamesPattern = gitLogTagNamesPattern;
         this.gitLogGroupByVersions = gitLogGroupByVersions;
+        this.gitLogJiraInstanceBaseUrl = gitLogJiraInstanceBaseUrl;
         this.log = log;
     }
 
-    public static String formatJiraIssuesToString(Collection<String> jiraIssues) {
+    public String formatJiraIssuesToString(Collection<String> jiraIssues) {
 
         StringBuilder output = new StringBuilder(100);
 
         for (String jiraIssueKey : jiraIssues) {
-            output.append("{jira:").append(jiraIssueKey).append("}\\\\\n");
+            output.append("{jira:").append(jiraIssueKey);
+
+            if (gitLogJiraInstanceBaseUrl != null) {
+                output.append("|url=").append(gitLogJiraInstanceBaseUrl).append("/browse/").append(jiraIssueKey);
```

Comment:
I have to use the url param for the jira issue instead of the baseurl param (https://confluence.atlassian.com/doc/jira-issues-macro-139380.html). The baseurl param worked for the link of the issue but not for the very helpful preview.